easybuild/easyconfigs/k/Keras_Applications/Keras_Applications-1.0.6-CrayGNU-18.08-pyhton3.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = "PythonPackage"
+
+name = 'Keras_Applications'
+version = '1.0.6'
+
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '5.1'
+
+pyver = '%s.%s' % (py_maj_ver, py_min_ver)
```

Comment:
Could you update `pyver` to the [new style](https://github.com/eth-cscs/production/wiki/UES-standards-(CUDA,-Python,-...)-for-EB-recipes#python)?
```python
pyver      = "%s.%s.%s" % (py_maj_ver, py_min_ver, py_rev_ver)
pyshortver = "%s.%s" % (py_maj_ver, py_min_ver)
```